airqo-platform/AirQo-api
# AN-297-Adding unit tests for Device Registry.

**_WHAT DOES THIS PR DO?_**
- Adding Unit Tests for device registry.

**_STATUS OF CODE COMPLETION_**
   
   - [x] create-activity
   - [x] create-airqloud
   - [x] create-device
   - [ ] create-event
   - [x] create-health-tips
   - [x] create-location
   - [x] create-monitor
   - [x] create-photo
   - [x] create-sensor
   - [x] create-site
   - [x] date
   - [x] distance
   - [x] errors
   - [x] generate Filter
   - [x] log
   - [x] multitenancy
  
**_STATUS OF REVIEW_**
   - [ ] create-activity
   - [ ] create-airqloud
   - [ ] create-device
   - [ ] create-event
   - [ ] create-health-tips
   - [ ] create-location
   - [ ] create-monitor
   - [ ] create-photo
   - [ ] create-sensor
   - [ ] create-site
   - [ ] date
   - [ ] distance
   - [ ] errors
   - [ ] generate Filter
   - [ ] log
   - [ ] multitenancy


**_WHAT ISSUES ARE RELATED TO THIS PR?_**

- Jira cards
    - [AN-297](https://airqoteam.atlassian.net/browse/AN-297)

**_HOW DO I TEST OUT THIS PR?_**
- Check out this branch.
- Run npm install
- run `mocha ./utils/test/ut_create-site.js --exit`
-You can currently test out the util tests depending on their indicated status of completion above. Simply replace the `ut_create-site.js` in the command below with the targeted test-file for the specific test, for-example `ut_create-device.js`

**_WHICH ENDPOINTS SHOULD BE READY FOR TESTING?:_** 

**_IS THERE ANY JENKINS CONSOLE LINK FOR CODE COVERAGE AND BUILD INFO?_**

**_ARE THERE ANY RELATED PRs?_**




[AN-297]: https://airqoteam.atlassian.net/browse/AN-297?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
